src/Docker.DotNet/Microsoft.Net.Http.Client/ManagedHandler.cs

```diff
@@ -156,7 +156,7 @@ private bool IsAllowedRedirectResponse(HttpRequestMessage request, HttpResponseM
                 if (_socketOpener != null)
```

Comment:
It indeed is, Add ?w=1 to ignore whitespace (so you can see the difference) https://github.com/Microsoft/Docker.DotNet/pull/234/files/c2456a2a599b914ac7b4047ef9652b575220bf95?w=1